Scripts/Services/PersonalityInsights/v3/DataModels.cs

```diff
@@ -0,0 +1,272 @@
+	{
+		/// <summary>
+		/// The identifier of the warning message, one of `WORD_COUNT_MESSAGE`, `JSON_AS_TEXT`,
+		/// or `PARTIAL_TEXT_USED`. 
+		/// </summary>
+		public string warning_id { get; set; }
+		/// <summary>
+		/// The message associated with the `warning_id`. For `WORD_COUNT_MESSAGE`, "There were 
+		/// <number> words in the input. We need a minimum of 600, preferably 1,200 or more, to 
+		/// compute statistically significant estimates."; for `JSON_AS_TEXT`, "Request input
+		/// was processed as text/plain as indicated, however detected a JSON input. Did you 
+		/// mean application/json?"; and for `PARTIAL_TEXT_USED`, "The text provided to compute the 
+		/// profile was trimmed for performance reasons. This action does not affect the accuracy
+		/// of the output, as not all of the input text was required." The `PARTIAL_TEXT_USED`
+		/// warning applies only when Arabic input text exceeds a threshold at which additional 
+		/// words do not contribute to the accuracy of the profile.
+		/// </summary>
+		public string message { get; set; }
+	}
+
+	/// <summary>
+	/// The Consumption Preferences Node object.
+	/// </summary>
+	[SerializeField]
+	public class ConsumptionPreferencesNode
+	{
+		/// <summary>
+		/// The unique identifier of the consumption preference to which the results pertain.
+		/// IDs have the form `consumption_preferences_`. 
+		/// </summary>
+		public string consumption_preference_id { get; set; }
+		/// <summary>
+		/// The user-visible name of the consumption preference. 
+		/// </summary>
+		public string name { get; set; }
+		/// <summary>
+		/// The score for the consumption preference: `0.0` indicates unlikely, `0.5` indicates
+		/// neutrality, and `1.0` indicates likely. The scores for some preferences are binary and
+		/// do not allow a neutral value. The score is an indication of preference based on the
+		/// results inferred from the input text, not a normalized percentile.
+		/// </summary>
+		public double score { get; set; }
+	}
+
+	/// <summary>
+	/// The content type. Either text, html or json.
+	/// </summary>
+	public class ContentType
+	{
+		/// <summary>
+		/// Mime type for plain text.
+		/// </summary>
+		public const string TEXT_PLAIN = "text/plain";
+
+		/// <summary>
+		/// Mime type for HTML.
+		/// </summary>
+		public const string TEXT_HTML = "text/html";
+
+		/// <summary>
+		/// Mime type for json.
+		/// </summary>
+		public const string APPLICATION_JSON = "application/json";
+	}
+
+	/// <summary>
+	/// The content language. Either English, Arabic, Spanish or Japanese.
+	/// </summary>
+	public class Language
+	{
+		/// <summary>
+		/// English.
+		/// </summary>
+		public const string ENGLISH = "en";
+		/// <summary>
+		/// Arabic.
+		/// </summary>
+		public const string ARABIC = "ar";
+		/// <summary>
+		/// Spanish.
+		/// </summary>
+		public const string SPANISH = "es";
+		/// <summary>
+		/// Japanese
+		/// </summary>
+		public const string JAPANESE = "ja";
```

Comment:
is `Content-Language` in the dropdown not accurate?
https://watson-api-explorer.mybluemix.net/apis/personality-insights-v3#!/personality-insights/profile